src/org/daisy/dotify/formatter/impl/segment/MarkerReferenceSegment.java

```diff
@@ -0,0 +1,93 @@
+package org.daisy.dotify.formatter.impl.segment;
+
+import org.daisy.dotify.api.formatter.MarkerReference;
+import org.daisy.dotify.api.formatter.TextProperties;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+/**
+ * Provides a marker reference event object.
+ */
+public class MarkerReferenceSegment implements Segment {
+
+    private final MarkerReference ref;
+    private final TextProperties props;
+    private Function<MarkerReference, String> v = (x) -> "";
```

Comment:
Please use a better description instead of `v`
e.g. `resolver`

Reply:
Done.